src/elements/paginator/paginator.ts

```diff
@@ -192,77 +190,46 @@ export class SbbPaginatorElement extends SbbNegativeMixin(LitElement) {
     }
 
-    switch (event.key) {
-      case 'Home':
-      case 'PageUp':
-        this._moveToPage(event, 0);
-        break;
-
-      case 'End':
-      case 'PageDown':
-        this._moveToPage(event, this._getVisiblePages().length - 1);
-        break;
-
-      case ' ': {
-        event.preventDefault();
-        const current = this._getVisiblePages().find((e: Element) => e === event.target);
-        if (current) {
-          this.pageIndex = +current.getAttribute('data-index')!;
-          this._markForFocus = this.pageIndex;
-        }
-        break;
-      }
+    event.preventDefault();
+    const current = this._getVisiblePages().find((e: Element) => e === event.target);
+    if (current) {
+      this.pageIndex = +current.getAttribute('data-index')!;
+      this._markForFocus = this.pageIndex;
     }
   }
 
   private _renderPrevNextButtons(): TemplateResult {
     return html`
-      <div class="sbb-paginator__buttons">
-        <sbb-mini-button-group ?negative=${this.negative} size=${this.size === 's' ? 's' : 'l'}>
-          <sbb-mini-button
-            id="sbb-paginator-prev-page"
-            aria-label=${i18nPreviousPage[this._language.current]}
-            icon-name="chevron-small-left-small"
-            ?disabled=${this.pageIndex === 0}
-            @click=${() => this._changePage(this.pageIndex - 1)}
-          ></sbb-mini-button>
-          <sbb-divider orientation="vertical"></sbb-divider>
-          <sbb-mini-button
-            id="sbb-paginator-next-page"
-            aria-label=${i18nNextPage[this._language.current]}
-            icon-name="chevron-small-right-small"
-            ?disabled=${this.pageIndex === this._numberOfPages - 1}
-            @click=${() => this._changePage(this.pageIndex + 1)}
-          ></sbb-mini-button>
-        </sbb-mini-button-group>
-      </div>
+      <sbb-mini-button-group ?negative=${this.negative} size=${this.size === 's' ? 's' : 'l'}>
+        <sbb-mini-button
+          id="sbb-paginator-prev-page"
+          aria-label=${i18nPreviousPage[this._language.current]}
+          icon-name="chevron-small-left-small"
+          ?disabled=${this.pageIndex === 0}
+          @click=${() => (this.pageIndex -= 1)}
+        ></sbb-mini-button>
+        <sbb-divider orientation="vertical"></sbb-divider>
+        <sbb-mini-button
+          id="sbb-paginator-next-page"
+          aria-label=${i18nNextPage[this._language.current]}
+          icon-name="chevron-small-right-small"
+          ?disabled=${this.pageIndex === this._numberOfPages - 1}
+          @click=${() => (this.pageIndex += 1)}
+        ></sbb-mini-button>
+      </sbb-mini-button-group>
     `;
   }
 
   private _renderItemPerPageTemplate(): TemplateResult | typeof nothing {
     return this.pageSizeOptions && this.pageSizeOptions.length > 0
       ? html`
           <div class="sbb-paginator__page-size-options">
-            Items per page
+            <label>${i18nItemsPerPage[this._language.current]}</label>
```

Comment:
The missing label was noticed by Mürsel, I tried to link it with the sbb-select via for/id but it does not work (the label should be inside the form-field).
If wanted, i could add a click handler to open/close the select.. what do you think?

Reply:
Not sure. we should implement native form support for select and check then? or add a temporary solution for now? @kyubisation Please help us here to decide :)

Reply:
As a side note, Mursel also wrote me that:

'The label of the combo box "Items per page" is not associated with this element. Therefore the combobox is output only with the value, but no label'

I have to test it with the new label tag, not sure if it will solve

Reply:
what if you wrap the label around the select?

Reply:
other idea. the combox of the select is anyways put into the light DOM. If you could apply an id to it, it could be linked with the label. the id creation would need some refactoring of the select, but should be solvable

Reply:
See commit 8291b28.

apparently, the `aria-labelledby` on the sbb-select does not work, and it could be tested comparing the native select in the form-field [here](https://lyne-storybook.app.sbb.ch/?path=/story/elements-sbb-form-field-sbb-form-field--select) with the sbb-select [here](https://lyne-storybook.app.sbb.ch/?path=/story/elements-sbb-select--single-select&args=value:Option+1;placeholder:): for the first one Voiceover reads correctly the value, the label and the fact that is a combobox, while for the second one it never reads the label.

To fix this and to make the paginator work with the external label, i added a `MutationObserver` in the select with the duty to proxy the `aria-labelledby` attribute to the trigger element, then in the paginator i used the `firstUpdated` hook to set the attribute.